bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Conjunction/Generator.cpp

```diff
@@ -0,0 +1,142 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Astrodynamics/Conjunction/Generator/Generator.hpp>
+
+inline void OpenSpaceToolkitAstrodynamicsPy_Conjunction_Generator(pybind11::module& aModule)
+{
+    using namespace pybind11;
+
+    using ostk::core::container::Array;
+
+    using ostk::physics::time::Duration;
+    using ostk::physics::time::Interval;
+
+    using ostk::astrodynamics::conjunction::CloseApproach;
+    using ostk::astrodynamics::conjunction::Generator;
+    using ostk::astrodynamics::Trajectory;
+
+    class_<Generator>(
+        aModule,
+        "Generator",
+        R"doc(
+            Find close approaches to a reference trajectory.
+
+            This class finds close approach events between a reference trajectory and other object trajectories
+            over a specified time interval. It uses a temporal condition solver to identify time periods when objects
+            are approaching and then determines the exact time of closest approach.
+        )doc"
+    )
+
+        .def(
+            init<const Trajectory&, const Duration&, const Duration&>(),
+            R"doc(
+                Constructor.
+
+                Args:
+                    reference_trajectory (Trajectory): The reference trajectory for which to find close approaches (Object 1).
+                    search_step_size (Duration): The step size to use during the close approach search. Defaults to Duration.seconds(15.0).
+                    search_tolerance (Duration): The tolerance to use during the close approach search. Defaults to Duration.milliseconds(1.0).
```

Comment:
```suggestion
                    step (Duration): The step size to use during the close approach search. Defaults to Duration.seconds(15.0).
                    tolerance (Duration): The tolerance to use during the close approach search. Defaults to Duration.milliseconds(1.0).
```
To be consistent with the access generator.